openpype/settings/defaults/project_anatomy/attributes.json

```diff
@@ -10,16 +10,8 @@
     "resolutionHeight": 1080,
     "pixelAspect": 1.0,
     "applications": [
-        "maya/2020",
-        "nuke/12-2",
-        "nukex/12-2",
-        "hiero/12-2",
-        "resolve/stable",
-        "houdini/18-5",
-        "blender/2-91",
-        "harmony/20",
-        "photoshop/2021",
-        "aftereffects/2021"
+        "maya/2024",
+        "nuke/14-0"
```

Comment:
This suddenly seems like a **very limited default** with only maya and nuke visible for new projects for new users of this repository?

Reply:
or better, add latest version of everything we support in Applications addon defaults and turn on "show only available"

Reply:
Agreed 100% with Ondrej here